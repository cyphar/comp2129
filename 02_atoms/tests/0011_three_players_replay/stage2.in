LOAD stage0.as
PLAYFROM 0
DISPLAY
PLACE 1 1
PLACE 4 2
PLACE 4 1
PLACE 3 0
PLACE 0 0
PLACE 2 1
PLACE 0 1
PLACE 2 1
DISPLAY
PLACE 4 1
PLACE 3 1
PLACE 4 1
PLACE 0 1
DISPLAY
STAT
PLACE 0 0
HELP
UNDO
PLACE 0 0
DISPLAY
PLACE 1 0
PLACE 0 0
STAT
DISPLAY
PLACE 0 1
PLACE 1 0
