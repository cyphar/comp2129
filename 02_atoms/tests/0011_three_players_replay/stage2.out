Game Loaded

Game Ready
Red's Turn


+--------------+
|  |  |  |  |  |
|  |  |  |  |  |
+--------------+

Green's Turn

Invalid Coordinates

Purple's Turn

Red's Turn

Green's Turn

Purple's Turn

Red's Turn

Cannot Place Atom Here


+--------------+
|R1|  |  |P1|  |
|P1|R1|G1|  |G1|
+--------------+

Cannot Place Atom Here

Green's Turn

Purple's Turn

Green's Turn


+--------------+
|  |P1|  |P1|G1|
|P1|P2|G1|G2|  |
+--------------+

Player Red:
Lost

Player Green:
Grid Count: 3

Player Purple:
Grid Count: 4

Purple's Turn


HELP displays this help message
QUIT quits the current game

DISPLAY draws the game board in terminal
START <number of players> <width> <height> starts the game
PLACE <x> <y> places an atom in a grid space
UNDO undoes the last move made
STAT displays game statistics

SAVE <filename> saves the state of the game
LOAD <filename> loads a save file
PLAYFROM <turn> plays from n steps into the game

Green's Turn

Purple's Turn


+--------------+
|G1|P1|  |P1|G1|
|P1|P2|G1|G2|  |
+--------------+

Green's Turn

Purple's Turn

Player Red:
Lost

Player Green:
Grid Count: 7

Player Purple:
Grid Count: 1


+--------------+
|G1|G2|G1|P1|G1|
|  |G2|G2|G2|  |
+--------------+

Green's Turn

Green Wins!
